### PR TITLE
Modernized the acceptance test template

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: tests/acceptance/README [hunk 1]
```diff
@@ -325,16 +324,30 @@ bundle agent test
...
+
+  # Consider using one of the dcs bundles
+  # methods: "Pass/Fail" usebundle => dcs_passif( "Namespace_Scoped_Class_Indicating_Test_Passed", $(this.promise_filename) );
+
+}
+bundle agent __main__
+{
+  methods:
+      "Run the default test bundle sequence (init,test,check,cleanup) if defined"
+        usebundle => default( $(this.promise_filename) );
```
Comment (olehermanse) [inline tests/acceptance/README:351]: I would maybe add a comment saying that the testall script actually runs the agent with the acceptance test as the entry point:

```
cf-agent -f ./99_path/to_test.cf
```

(This was not obvious to me for a while).

**Review by olehermanse — DISMISSED** [review]
Great!

**Comment by larsewi** [comment]
Nice!

File: tests/acceptance/README (hunk 1 above)
Comment (nickanderson) [reply tests/acceptance/README]: Ack, added, what do you think?

File: tests/acceptance/README [hunk 2]
```diff
@@ -345,6 +345,17 @@ bundle agent check
 
 }
 bundle agent __main__
+# @brief The test entry
+# Note: The testall script runs the agent with the acceptance test as the entry point,
+##+begin_src sh :results output :exports both :dir ~/CFEngine/core/tests/acceptance
+# pwd | sed 's/\/.*CFEngine\///'
+# find . -name "*.cf*" | xargs chmod 600
+# cf-agent -Kf ./01_vars/02_functions/basename_1.cf --define AUTO
+##+end_src sh
```
Comment (olehermanse) [inline tests/acceptance/README:354]: org makes it ugly 😢

**Review by olehermanse — APPROVED** [review]

**Pull request merged**